modules/rosa-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-creation-and-sharing_{context}"]
+= Configuring a VPC to share within your AWS organization
+
+You can share your configured VPC with another AWS user account if that account is within your current AWS organization.
+
+.Procedure
+
+. On your VPC-owning AWS account ID, you must create your VPC to your desired specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console]. 
+. In the link:https://us-east-1.console.aws.amazon.com/iamv2/[Identity and Access Management (IAM) section of the AWS console], create a new custom trust policy role for the shared VPC permissions. This role should have the following permissions:
+  * A trust policy to assume roles:
++
+[source,terminal]
+----
+{
+	"Version": "2012-10-17",
+	"Statement": [
+		{
+			"Sid": "Statement1",
+			"Effect": "Allow",
+			"Principal": {}, <1>
+			"Action": "sts:AssumeRole"
+		}
+	]
+}
+----
++
+--
+<1> The principals are added later in this process once the cluster-creating AWS account user has created these roles.
+--
+    * `ResourceGroupandTagEditorFullAccess` permissions policy
+    * `Route53minimalPermissions` permissions policy
++
+After you create this IAM role, you must give the created role's ARN to the cluster creator.
+. In the link:https://us-east-1.console.aws.amazon.com/ram/[Resource Access Manager of the AWS console], create a Resource Share that includes your VPC's private and public subnets and the cluster creator's AWS account ID.
+
+After creating this Resource Share, the cluster-creating AWS account user must create a DNS domain and operator roles to continue configuration.
```

Comment:
I think it might be clearer if the two separate users were always referred to in a similar manner. For example, in this sentence you write "cluster-creating AWS account user" while in the previous sentence it reads "the cluster creator".